<?xml version='1.0'?>
<launch>

    <arg name="joy_topic_name" default="joy" />

    <node pkg="joy" type="joy_node" name="joy">
        <param name="autorepeat_rate" value="10" />
        <param name="dev" value="/dev/input/js0" />
    </node>

    <node pkg="rover_control" name="teleop_joy" type="teleop_joystick" output="screen">
        <rosparam file="$(find rover_control)/config/joy_keybind.yaml"/>
        <!-- <remap from="joy" to="$(arg joy_topic_name)" /> -->
    </node>
</launch>
